docs/source/welcome_to_sycamore/get_started.md

```diff
@@ -121,6 +119,50 @@ This will crawl and download the data from the specified website. If you import
 If you want to prepare you data with custom code, you can use a Jupyter notebook to iterate and test it. NEED LINK
 
 
+## Add a dataset from a S3 bucket
+
+You can try using an arbitrary PRF and HTML data from an S3 bucket with the Sort Benchmark
+importing script. This script is not optimized for new datasets, so the answer quality may
```

Comment:
for arbitrary datasets?  (The problem isn't the new, but the arbitrary, I'd expect it to work pretty well on the VLDB conference pdfs.)

Reply:
Ack.